config.py

```diff
@@ -6,3 +6,10 @@
 DEBUG = os.environ.get('DEBUG', 0)
 SQLALCHEMY_DATABASE_URI = os.environ['DATABASE_URL']
 SQLALCHEMY_MIGRATE_REPO = os.path.join(basedir, 'db_repository')
+
+BUTTON_ACTIONS = [
```

Comment:
This is a discussion we should take as a group too, considering I'd personally prefer just saying "fast, slow, easy, hard", but considering this is a "bot", we might want to be more communicative with the user.